lib/stdio/utils.js

```diff
@@ -7,3 +8,16 @@ export const isBinary = value => isUint8Array(value) || Buffer.isBuffer(value);
 
 const textDecoder = new TextDecoder();
 export const binaryToString = uint8ArrayOrBuffer => textDecoder.decode(uint8ArrayOrBuffer);
+
+// Like `source.pipe(destination)`, if `source` ends, `destination` ends.
+// Like `stream.pipeline(source, destination)`, if `source` aborts/errors, `destination` aborts.
```

Comment:
```suggestion
// Like `Stream.pipeline(source, destination)`, if `source` aborts/errors, `destination` aborts.
```

Otherwise it reads as if it's `Stream.prototype.pipeline`.